posthog/api/decide.py

```diff
@@ -64,12 +65,15 @@ def on_permitted_recording_domain(team: Team, request: HttpRequest) -> bool:
     return is_authorized_web_client or is_authorized_mobile_client
 
 
-@csrf_exempt
-@timed("posthog_cloud_decide_endpoint")
-def get_decide(request: HttpRequest):
-    # handle cors request
-    if request.method == "OPTIONS":
-        return cors_response(request, JsonResponse({"status": 1}))
+def get_base_config(token: str, team: Team, request: HttpRequest, skip_db: bool = False) -> dict:
+    if token in (settings.DECIDE_TOKENS_FOR_REMOTE_CONFIG or []):
+        response = RemoteConfig.get_config_via_token(token)
```

Comment:
should this be wrapped in a try/catch in case the remote config fails to connect to the cache for some reason?

Reply:
It _probably_ should although this isn't the final step. This allows me to test against prod for parity. Once I've ironed our the issues then I will add some more metrics and fallbacks.

That said, once we move over to it it should be _as_ reliable as the existing thing as it uses the same cache access concept.